### PR TITLE
Add example: who still win drivers WDC

### PR DESCRIPTION
**Pull request by oscr**
Suggesting a relatively "simple" example for people getting started. If/when #229 is implemented this example can be simplified and improved.

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -10,4 +10,5 @@ wheel
 flake8==3.9.2
 readme-renderer
 xdoctest
-pre-commit
\ No newline at end of file
+pre-commit
+requests==2.28.1
```
Comment (theOehrly) [inline requirements-dev.txt:14]: Why are you pinning requests here?

**Comment by theOehrly** [comment]
Overall, this is cool. But the example is not rendered correctly in the documentation. 
The examples gallery is created from these scripts and then shown here: https://theoehrly.github.io/Fast-F1/examples_gallery/index.html
Therefore, the scripts need to be formatted so that Sphinx-Gallery (https://sphinx-gallery.github.io/stable/index.html) can read them and generate the associated page for the docs.
The script should also produce an example output, imo. It doesn't need to be a matplotlib plot, though. You can take a look at the "Qualifying Results Overview" example in which there is some intermediary output.

If you don't have a local environment to build the docs, you can download the CI build as an artifact after the checks have run here.

**Comment by oscr** [comment]
I've been able to get `make singlehtml` working. But how does output work? For example here: https://theoehrly.github.io/Fast-F1/examples_gallery/plot_qualifying_results.html#qualifying-results-overview looking at the yellow output field there doesn't seem to be any special markdown in the code. How is this generated?

**Comment by theOehrly** [comment]
> I've been able to get `make singlehtml` working. But how does output work? For example here: https://theoehrly.github.io/Fast-F1/examples_gallery/plot_qualifying_results.html#qualifying-results-overview looking at the yellow output field there doesn't seem to be any special markdown in the code. How is this generated?

Everything that is output on stdout is printed like this if I remember correctly.

File: requirements-dev.txt (hunk 1 above)
Comment (theOehrly) [reply requirements-dev.txt:14]: `requests` is already a requirement in setup.cfg so it is maybe not necessary to add it here. Just that I don't have the version pinned there. 
Was your intention to specifically pin the version or did you just want to add it in general?

**Comment by oscr** [comment]
I was about to give up totally when I found it: 

```
gallery_conf["filename_pattern"] = '/plot'
```

Renamed the example and it worked.

File: requirements-dev.txt (hunk 1 above)
Comment (oscr) [reply requirements-dev.txt]: I'm not so good at Python packaging and I just added it because IDEA complained. I removed it again.

**Comment by theOehrly** [comment]
> I was about to give up totally when I found it:
> 
> ```
> gallery_conf["filename_pattern"] = '/plot'
> ```
> 
> Renamed the example and it worked.

When you're stuck on things like this, feel free to ask instead of wasting too much time on it.

**Comment by oscr** [comment]
Awesome! Thank you for your help Philipp!

**Pull request merged**